### DISCUSSION TITLE
Is there a way to make it work with *.scala.html (Playframework's Scala templates)?

### DISCUSSION THREAD

**Discussion by tanin47** [scalameta/metals#5993] [General]
Otherwise, everything works pretty well on VSCode. Only *.scala.html doesn't Thank you.

**Comment by tgodzik** [comment]
It's currently not supported, would be awesome if anybody had a stab at it, but we need to focus more on stability currently or features improving that.

https://github.com/scalameta/metals-feature-requests/issues/50
